Build error after upgrading to Angular 10 and opting out of Ivy

## Library
- `msal@1.4.0`
- `@azure/msal-angular@1.1.1`
- `@angular/core@10.1.1`

Disabled Ivy: `"angularCompilerOptions": { "enableIvy": false }`
## Framework
Angular 10

## Description
After upgrading to angular 10 MSAL gives us a build error.
We were using older versions of MSAL before the upgrade and figured updating to the latest versions would fix something.
This was not the case though and looking at the NPM page implies that angular 10 is not supported:
![image](https://user-images.githubusercontent.com/64589372/93433370-fff9de80-f8c6-11ea-81f7-603ea9fff8da.png)
However, there's also a line about following the angular support schedule:
![image](https://user-images.githubusercontent.com/64589372/93433455-1bfd8000-f8c7-11ea-8042-8641381bc3ca.png)
To me, this implies that angular 10 is supported, since 10 is currently active.
![image](https://user-images.githubusercontent.com/64589372/93433522-2e77b980-f8c7-11ea-9332-c282d196d95c.png)

With a build error like it leads me to conclude that angular 10 is not supported by the msal-angular package.
Is this conclusion correct?

## Error Message
`msal.module.d.ts:5:86 - error TS2314: Generic type 'ModuleWithProviders<T>' requires 1 type argument(s)`
Importing ModuleWithProviders from angular core: `import { ModuleWithProviders } from "@angular/core";`

--- a/change/@azure-msal-angular-2020-11-10-12-09-45-angular-v1-peer-dep.json
+++ b/change/@azure-msal-angular-2020-11-10-12-09-45-angular-v1-peer-dep.json
@@ -1,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Pass generic to ModuleWithProviders for MsalModule v1",
+  "packageName": "@azure/msal-angular",
+  "email": "janutter@microsoft.com",
+  "dependentChangeType": "patch",
+  "date": "2020-11-10T20:09:45.661Z"
+}

--- a/change/@azure-msal-angular-2020-11-10-12-09-45-angular-v1-peer-dep.json
+++ b/change/@azure-msal-angular-2020-11-10-12-09-45-angular-v1-peer-dep.json
@@ -1,8 +1,8 @@
 {
   "type": "patch",
-  "comment": "Pass generic to ModuleWithProviders for MsalModule v1",
+  "comment": "Pass generic to ModuleWithProviders for MsalModule v1, set supported Angular versions to 6-9 (#2577)",
   "packageName": "@azure/msal-angular",
   "email": "janutter@microsoft.com",
-  "dependentChangeType": "patch",
+  "dependentChangeType": "none",
   "date": "2020-11-10T20:09:45.661Z"
 }

--- a/lib/msal-angular/package.json
+++ b/lib/msal-angular/package.json
@@ -33,8 +33,8 @@
     ]
   },
   "peerDependencies": {
-    "@angular/common": ">= 6.0.0",
-    "@angular/core": ">= 6.0.0",
+    "@angular/common": "6 - 9",
+    "@angular/core": "6 - 9",
     "msal": "^1.4.3",
     "rxjs": "^6.0.0"
   },

--- a/lib/msal-angular/src/msal.module.ts
+++ b/lib/msal-angular/src/msal.module.ts
@@ -23,7 +23,7 @@ export class MsalModule {
     static forRoot(
         config: Configuration,
         angularConfig: MsalAngularConfiguration = defaultMsalAngularConfiguration
-    ): ModuleWithProviders {
+    ): ModuleWithProviders<MsalModule> {
         return {
             ngModule: MsalModule,
             providers: [

